packages/gatsby-plugin-manifest/README.md

```diff
@@ -19,6 +19,11 @@ the created manifest.json.
 
 ## How to use
 
+There are three configurations in which this plugin will function: manual, hybrid, and auto. What config options you include will determine how the plugin functions.
```

Comment:
I'd stay consistent with naming things, so either auto here and in the header below, or automatic in both cases. Not a huge deal, just makes it easier to scan the document for relevant information.

Might also make sense to stick with config or configuration and stay consistent.

Reply:
Also, will people know generally what this plugin is about? A brief description before the how to use section could be handy.

Reply:
What about also rewording the last sentence to read "These three configuration options are explained below. The plugin functions differently depending on which of the three you choose." This would clearly link this paragraph with the following paragraph, and linking is nearly always a great thing.

Reply:
All sound good